### ISSUE TITLE
Open the ProgCode bank account in a credit union / a community-driven bank.

### ISSUE THREAD

**Issue by rcscastillo** [ProgressiveCoders/functions#191]
#### Description

We need to support the community driven efforts as a reflection of who we are as an organization.

#### Problem

We need to open up a bank account for the organization, and in order to do that we need to support credit unions. 

#### Benefit

Support local and grassroots effort for sustainability.

#### Plan

- [ ] Consent Vote
- [ ] Community Blessing
- [ ] Discuss What credit union
- [ ] Vote on that specific credit union

#### Decision Making

By consent

#### Reference link(s)

...

**Comment by jpb5013** [comment]
Love it!

**Comment by desmondjones** [comment]
Perfect.  It would be helpful, too, to tie it to a electronic funds service such as PayPal.

**Comment by WoobieTuesday** [comment]
When we say "community driven efforts" are we specifically referring to NYC?

**Comment by rapicastillo** [comment]
No. Community driven efforts are the credit union setup by communities.
